test/boskos.sh

```diff
@@ -1,5 +1,24 @@
 #!/usr/bin/env bash
 
+# Copyright 2024 The Kubernetes Authors.
```

Comment:
we should be able to leave this whole thing to the kubetest2 deployer? but let's leave that to a follow-up

Reply:
I had problems leaving it up to `kubetest2` to create the boskos project, because I needed to pass the project name on to the `ginkgo-tester` test args.